### PR TITLE
Enable vcr for metric specs

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/143/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/986/ : SUCCESS in 1m 13s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1255/ : SUCCESS in 2m 32s

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -80,6 +80,14 @@ def peach_with_index(*)
 
         # insert the cassete recording everything what happens outside the tests cases
         VCR.insert_cassette("#{self.class.metadata[:description]}/all")
+
+        # avoid polling idle time by overriding sleep
+        module Kernel
+          alias :old_sleep :sleep
+          def sleep(n)
```
Comment (yenkins) [inline spec/spec_helper.rb]: Has the parameter name 'n' (UncommunicativeParameterName)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/144/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/987/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1256/ : SUCCESS in 2m 11s

**Comment by panjan** [comment]
extended test

**Comment by kubamahnert** [comment]
ok to merge

**Pull request merged**